src/java/us/kbase/mobu/initializer/ModuleInitializer.java

```diff
@@ -153,7 +153,9 @@ public void initialize(boolean example) throws Exception {
             templateFiles.put("module_tox", Paths.get(moduleDir, "tox.ini"));
             break;
         case "perl":
-            templateFiles.put("module_test_perl_client", Paths.get(moduleDir, "test", this.moduleName + "_server_test.pl"));
+            templateFiles.put("module_cpanfile", Paths.get(moduleDir, "cpanfile"));
+            templateFiles.put("module_test_perl_client", Paths.get(moduleDir, "test", this.moduleName + "_impl.t"));
```

Comment:
rename - perl tests should have the `.t` suffix to be automatically picked up by the test harness